src/tensor_ops/accurate_gelu/cpu_kernel.rs

```diff
@@ -0,0 +1,55 @@
+use crate::tensor_ops::cpu_kernels::UnaryDerivative;
+#[cfg(feature = "f16")]
+use half::f16;
+use libm::{erf, erff};
+use num_traits::{Float, FloatConst};
+
+trait Erf {
+    fn erf(self) -> Self;
+}
+
+#[cfg(feature = "f16")]
+impl Erf for f16 {
+    fn erf(self) -> Self {
+        f16::from_f32(erff(f16::to_f32(self)))
+    }
+}
+
+impl Erf for f64 {
+    fn erf(self) -> Self {
+        erf(self)
+    }
+}
+
+impl Erf for f32 {
+    fn erf(self) -> Self {
+        erff(self)
+    }
+}
```

Comment:
I feel like these should be nearer to the Dtype trait, it feels a little odd putting them here, but I'm not that familiar with the code base so don't know the best place to put them yet.

Reply:
I actually like this location. Keeping things as close to where they are used is preferred IMO. Not sure if there's a standard practice for this though, but I like it 👍